README.md

```diff
@@ -1,2 +1,46 @@
-# ocds_countryCode_extension
-Add a countryCode field in Address to specify an ISO3166-1 alpha-2 country code.
+# Country code
+
+Adds a countryCode property in Address.
```

Comment:
```md
Adds a country code field to the address object.
```

For "property" vs "field", see https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/style_guide.html#word-choice

Reply:
Done